picard/file.py

```diff
@@ -557,7 +557,6 @@ def move(self, parent):
                 self.parent.remove_file(self)
             self.parent = parent
             self.parent.add_file(self)
-            self.acoustid_update()
```

Comment:
Where is this handled now?

Reply:
It isn't. Not really sure if it affected anything. If the analysis was pending, it already got cancelled above. If the analysis already finished, it wouldn't be impacted. On my testing everything worked fine and ended up ignoring it.

Reply:
Faster solution: calling acoustid_update to copy the metadata but skipping update_item.